source/opt/aggressive_dead_code_elim_pass.h

```diff
@@ -79,8 +86,8 @@ class AggressiveDCEPass : public MemPass {
   // Return true if all extensions in this module are supported by this pass.
   bool AllExtensionsSupported() const;
 
-  // Returns true if |inst| is dead.  An instruction is dead if its result id
-  // is used in decoration or debug instructions only.
+  // Returns true if the target of |inst| is dead.  An instruction is dead if
+  // its result id is used in decoration or debug instructions only.
```

Comment:
Should say that the instruction is assumed to be an OpName, OpMemberName or an annotation instruction.

Reply:
Will change.